packages/salesforcedx-vscode-apex-debugger/package.nls.json

```diff
@@ -1,7 +1,7 @@
 {
-  "user_id_filter_text": "A list of user IDs with requests whitelisted for debugging",
-  "request_type_filter_text": "A list of Apex request types whitelisted for debugging",
-  "entry_point_filter_text": "A regular expression matching entry points whitelisted for debugging",
+  "user_id_filter_text": "A list of user IDs with requests allowlisted for debugging",
```

Comment:
We could replace the use of `allowlisted` with just `allowed`

Reply:
Done!